pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,102 @@
+
+We're also writing voluntary OSS bi-weekly updates via GitHub Gists to speak about the latest updates made in the AsyncAPI Docs Ecosystem. Due to our commitment to investing time in gaining interest in our community and getting Google excited about us, we've made sure to maintain updates about our `Google Season of Docs 2022` application too! In fact, you can take a look at the latest three where we made said mentions here in [AsyncAPI Docs update (31 Jan - 11 Feb 2022)](https://gist.github.com/alequetzalli/94ca1ffb5d123b450501e40a4a3b56e2), [AsyncAPI Docs update (14 Feb - 25 Feb 2022)](https://gist.github.com/alequetzalli/d34e3aececa49d10d0ddb2dc9938b477), and [AsyncAPI Docs update (28 Feb - 11 March 2022)](https://gist.github.com/alequetzalli/8f449f731b919193f4101098a69da14d).
+
+### 📏 Measuring our Docs project’s success
+We will measure success in the Docs project partially by capturing specific feedback about the IA changes via our soon-to-come new [Docs Feedback card](https://github.com/asyncapi/website/issues/453). We need this specific and granular feedback to make sure we listen and make changes according to what the community requests from Docs. In previous AsyncAPI Docs Gist updates, we've mentioned that Design contributors were teaming with Docs on `/website`issue [#453](https://github.com/asyncapi/website/issues/453) for the ideation and development of our new **feedback card** that will be added at the bottom of each Docs page. What the community decided over the last 2 weeks was that the `Submit feedback` button in the card will publish the feedback anonymously via the AsyncAPI bot and create a new **GitHub Discussion** with said feedback:
+
+![A screenshot displaying the design of our new feedback card for receiving feedback on AsyncAPI Docs](/img/posts/gsod-2022/feedback-card.webp)
+
+![A screenshot of AsyncAPI GitHub Discussions for the Docs category](/img/posts/gsod-2022/docs-community-discussions.webp)  
+
+The other way we would consider the project successful is the number of our contributors and Docs PRs increased by even 2-5 more community members. Currently, a majority of our OSS contributor community focuses only on contributing code, but we would like to instill a greater interest in contributing to documentation that provides value for everyone.
+
+
+### ⌛ Timeline
+The project itself will take approximately 3-6 months to complete, depending on how many technical writers (TW) get involved. If we can get even just 2-4 TWs, then we can work on both migrating the CLI/Tools documentation and the Generator Docs. But if we can only get 1-2 folks, it would be hard to do more than the migration of CLI and Tools docs. Ideally, the timeline could look like the following:
```

Comment:
Would it make sense if an exact number of TWs are mentioned? Most orgs go with 2 TWs while some orgs like Wechaty have gone with multiple TWs in the past. A clear mention of the number of TWs will also make sense from planning perspective and give an assurance to Google that the funds are moving towards the right direction.

Reply:
I don't know if it makes sense to do so or not. Google is very vague about processes and they don't really tell you how many TWs they may "give" you... my first thought was that perhaps showing flexibility for any number of TWs might make it easier to place TWs with us. That's why you see I wrote above what items could be done depending on the amount of TWs "sent" to us.

This is also just a clarification note that is not asked for in the proposal template, so you would think it shouldn't affect to make a realistic clarifying annendum. 🙊 

Is Google is super strict and doesn't like proposals showcasing different scenarios like I did above.. maybe? Honestly it's your guess as much as mine is 😄 

I will change it however to just state our preferred request and it prob won't change anything (as in, our likelihood to get accepted 😂) but sounds good to me 😄✨✨